apps/docs/pages/guides/getting-started/tutorials/with-flutter.mdx

```diff
@@ -5,6 +5,15 @@ export const meta = {
   description: 'Learn how to use Supabase in your Flutter App.',
 }
 
+<div className="video-container">
+  <iframe
+    src="https://www.youtube-nocookie.com/embed/r7ysVtZ5Row"
+    frameBorder="1"
+    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+    allowFullScreen
+  ></iframe>
+</div>
+
```

Comment:
Then we no longer would need this here:

```suggestion
```

Reply:
Thanks @gregnr ! I wasn't aware of that! That looks fantastic!